src/components/editableTable/SingleEditableTableRow.js

```diff
@@ -133,69 +106,43 @@ export default function EditableTableRow({
         }
     };
 
-    const Cell = column => {
-        const Content = () =>
-            (editing && column.editable && editable) || (isNewRow && editing && column.required)
-                ? inputComponentFactory(item, column, handleValueChange, rest)
-                : displayComponentFactory(item, column);
-        if (!column.tooltip) {
-            return (
-                <Fragment key={`${column?.id}${item.id}`}>
-                    <TableCell id={column.type}>
-                        <>{Content()}</>
-                    </TableCell>
-                </Fragment>
-            );
-        }
-        return (
-            <Fragment key={`${column?.id}${item.id}`}>
-                <Tooltip title={column.tooltip(item) || ''}>
-                    <TableCell id={column.type} key={`${column?.id}${item.id}`}>
-                        {Content()}
-                    </TableCell>
-                </Tooltip>
-            </Fragment>
-        );
-    };
-
-    Cell.propTypes = {
-        column: PropTypes.shape({
-            tooltip: PropTypes.func,
-            id: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
-            editable: PropTypes.bool,
-            type: PropTypes.string,
-            required: PropTypes.bool
-        })
-    };
-
-    Cell.defaultProps = {
-        column: null
+    const handleEditButtonClick = () => {
+        setEditing(true);
     };
 
     return (
         <ClickAwayListener onClickAway={e => handleClickAway(e)}>
```

Comment:
Did you say in show n tell that you'd killed the ClickAwayListener? It's still here so just wonderin'

Reply:
Yeah I wasn't very clear there. I'd removed it from the original editable table as it was causing the row to immediately switch back to the editable state after clicking the cancel button.

Originally here I'd removed it from the  group edit table but I've been messing around with it this morning and now got it working in both, so I could add it back in. I'm wondering if having it as an option is inconsistent - should a table only be interacted with through the buttons? My main thinking (and why I removed it from the group table initially) is that because the group edit table will set isToBeSaved to false when setting editing to true (so if a row is coloured green and is to be saved, clicking the edit button will clear that to be saved flag and set editing to true), having listeners on the full row and the click away could pick up some unintended clicks and might not be totally clear to users what is happening. The buttons make it really explicit that something will happen and a state change will be triggered. 

Do you reckon the click away and on click row behaviours should be options for both? I can't decide...

I might just be over thinking this...

Reply:
i don't think the click away options should be default. i think the buttons are safer and clearer for now. 
I don't see any problem in having it as an option for advanced users in certain forms if it doesn't add too much complexity to maintaining the code.